.gitignore

```diff
@@ -17,3 +17,47 @@ tmp
 _yardoc
 
 package/*.tar.bz2
+
+# Travis CI
+Makefile
+Makefile.am
+Makefile.am.common
+Makefile.in
+aclocal.m4
+agents/Makefile
+agents/Makefile.in
+autodocs-cc-base.ami
+autodocs-cc-off.ami
+autodocs-cc.ami
+autodocs-ycp.ami
+autom4te.cache/output.0
```

Comment:
You can add only autom4te.cache and it will ignore all files in that directory. The same applies to doc/.